src/main/java/seedu/address/model/ReadOnlyUserPrefs.java

```diff
@@ -13,8 +13,12 @@
     Path getAddressBookFilePath();
     String[] getBlockSettings();
     int getMinRoomSettings();
+    void setMinRoomSettings(int minRoomSettings);
     int getMaxRoomSettings();
+    void setMaxRoomSettings(int maxRoomSettings);
     int getMinFloorSettings();
+    void setMinFloorSettings(int minFloorSettings);
     int getMaxFloorSettings();
+    void setMaxFloorSettings(int maxFloorSettings);
```

Comment:
i think set functions violates the `ReadOnly` function of this class. Can consider using the `UserPrefs' class instead

Reply:
Good point, fixed.